test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+# set path for test directory
+TESTS_PATH=./test
```

Comment:
Di you consider using travis for environment variables? (Although it may make our travis file too verbose)

Reply:
I did not in this case. This variable is used only within this script so I found it more clear to keep it there